restapi/src/main/java/io/stargate/web/docsapi/service/DocumentService.java

```diff
@@ -999,8 +993,6 @@ void addRowsToMap(Map<String, List<Row>> rowsByDoc, List<Row> rows) {
     List<BuiltCondition> predicate =
         ImmutableList.of(BuiltCondition.of("key", Predicate.IN, new ArrayList<>(docNames)));
 
-    db = dbFactory.getDocDataStoreForToken(authToken, headers);
```

Comment:
Was this leftover from something else? Or is there some other reason?

Reply:
Yea this was just left over and didn't need to be here